frontend/src/pages/chat/Chat.module.css

```diff
@@ -70,15 +57,12 @@
 }
 
 .chatMessageStream {
-  flex-grow: 1;
-  max-width: 1028px;
-  width: 100%;
-  overflow-y: auto;
-  padding-left: 24px;
-  padding-right: 24px;
+  flex: 1;
   display: flex;
   flex-direction: column;
+  overflow-y: auto;
   margin-top: 24px;
+  padding: 0 max(calc(20px + 24px), calc((100% - var(--chat-max-width)) / 2));
```

Comment:
[dust] I'm usually anti-code comment but this formula seems like it could use some explanation so that someone could confidently edit it in the future. Or combo that with saving some of these magic numbers 20px and 24px into a named variable